src/main/java/org/broadinstitute/hellbender/tools/copynumber/denoising/HDF5SVDReadCountPanelOfNormals.java

```diff
@@ -305,32 +305,41 @@ public static void create(final File outFile,
             }
             logger.info(String.format("Performing SVD (truncated at %d eigensamples) of standardized counts (transposed to %d x %d)...",
                     numEigensamples, numPanelIntervals, numPanelSamples));
```

Comment:
Is there a test here for a single sample PoN?  Apologies if I missed it.

Reply:
Yup, I added single-sample tests, but also simplified how the GC/no-GC tests are run---so you might have missed it.